### PR TITLE
Adding maven files for Assesment-1

### PR DESCRIPTION
**Pull request by khushikdu**
Screenshots:
1. Menu
Screenshot from 2024-01-19 13-08-43

2. Displaying Bowlers with more than 40 wickets
Screenshot from 2024-01-19 13-08-58
Screenshot from 2024-01-19 13-09-02

3. Best Batsman and Bowler of a Team
Screenshot from 2024-01-19 13-09-20

4. Top 3 batsman and Bowler
Screenshot from 2024-01-19 13-09-35

5. Terminating the program
Screenshot from 2024-01-19 13-15-12
 
**Drift Link:** https://video.drift.com/v/abD6mvuOoPWKLkqhnCDMHXHNxndko3I007OQ5nmp8hn8/

### CODE REVIEW

**Comment by khushikdu** [comment]
| Metric | Value |
| ------ | ----- |
| alert_status | <b>:white_check_mark: OK</b> |
| bugs | 0 |
| code_smells | 12 |
| reliability_rating | A (0 Bugs)<br/>Note: A being best, E being worst |
| security_rating | A (0 Vulnerabilities)<br/>Note: A being best, E being worst |
| vulnerabilities | 0 |

File: backend/assessments/1/src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package org.example;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
```
Comment (Sagun15) [inline backend/assessments/1/src/main/java/org/example/Main.java:9]: Create separate package for it. And use generic functions to print logs

File: backend/assessments/1/src/main/java/org/example/Main.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    public static void main(String[] args) {
+        Scanner scanner=new Scanner(System.in);
+        Operations operations=new Operations();
+        List<Profile> profiles= operations.loadCSVData("src/main/resources/IPL_2021-data.csv");
```
Comment (Sagun15) [inline backend/assessments/1/src/main/java/org/example/Main.java:13]: Path hardcoded

File: backend/assessments/1/src/main/java/org/example/Main.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            int choice = scanner.nextInt();
+            try {
+
+                switch (choice) {
+                    case 1:
+                        List<Profile> bowler = operations.getBowler(profiles);
+                        logger.info("Bowlers having more than 40 wickets");
+                        for (Profile profile : bowler)
+                            logger.info("Name : {} | Wicktes : {} ", profile.getPlayerName(), profile.getWickets());
+                        break;
```
Comment (Sagun15) [inline backend/assessments/1/src/main/java/org/example/Main.java:28]: You didn't take user input for team name

File: backend/assessments/1/src/main/java/org/example/Main.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                    case 2:
+                        logger.info("Enter the team name : ");
+                        String teamName = scanner.next();
+                        profiles1 = operations.getBestBowler(profiles, teamName);
+                        if (profiles1 == null) {
+                            logger.info("Invalid team name");
+                            break;
+                        }
+                        logger.info("Best Bowler in {} is {} ", teamName, profiles1.get(0).getPlayerName());
+                        profiles1 = operations.getBestBatsman(profiles, teamName);
```
Comment (Sagun15) [inline backend/assessments/1/src/main/java/org/example/Main.java:39]: You did check for bowlers if it is null or not. But not for this case

File: backend/assessments/1/src/main/java/org/example/Main.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+                    case 2:
+                        logger.info("Enter the team name : ");
+                        String teamName = scanner.next();
+                        profiles1 = operations.getBestBowler(profiles, teamName);
+                        if (profiles1 == null) {
+                            logger.info("Invalid team name");
+                            break;
+                        }
+                        logger.info("Best Bowler in {} is {} ", teamName, profiles1.get(0).getPlayerName());
```
Comment (Sagun15) [inline backend/assessments/1/src/main/java/org/example/Main.java:38]: Other details were also expected to be logged

File: backend/assessments/1/src/main/java/org/example/Operations.java [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+        try {
+        return profiles.stream()
+                .filter(profile -> profile.getTeam().equalsIgnoreCase(team))
+                .sorted(Comparator.comparingDouble(Profile::getAverage).reversed())
+                .limit(1)
+                .collect(Collectors.toList());
+        }catch (Exception e){
+            e.printStackTrace();
+        }
+        return null;
```
Comment (Sagun15) [inline backend/assessments/1/src/main/java/org/example/Operations.java:85]: Missing java docs

**Pull request merged**